spec/serializers/npq_participant_serializer_spec.rb

```diff
@@ -4,9 +4,10 @@
 
 RSpec.describe NPQParticipantSerializer do
   describe "serialization" do
+    let!(:participant) { create(:user) }
```

Comment:
Don't need the bang here I don't think?

Reply:
yeah you are right, I just copypasta'd it from the other part of the spec